src/selectors/dispensary.js

```diff
@@ -67,3 +85,19 @@ export const selectSortedData = createSelector(
   [selectFilteredData, selectIsAscending, selectSortKey],
   (data, isAscending, sortKey) => data.sorted(sortKey, !isAscending)
 );
+
+export const selectLookupFormConfig = createSelector([selectDataSet], dataSource =>
+  getFormInputConfig(LOOKUP_FORM_CONFIGS[dataSource])
+);
+
+export const selectLookupListConfig = createSelector(
+  [selectDataSet, selectLookupFormConfig],
+  (dataSource, formConfig) => {
+    const listKeys = LOOKUP_LIST_CONFIGS[dataSource];
+    const listTypes = listKeys.reduce(
+      (acc, key) => ({ ...acc, [key]: formConfig.find(config => config.key === key)?.type }),
+      {}
+    );
+    return listKeys.map(key => ({ key, type: listTypes[key] }));
+  }
+);
```

Comment:
just because I often spend a lot of time with these decisions of DRY code: I'm not sure I like the coupling of these configs. I think you could be creating new column definitions and using `getColumns` or am I a bit off ?

Reply:
Definitely could, tbh the whole thing is probably a bit of a smell in general as it's a bit tricky to find the line between how much to use of the `formInputConfig` stuff and how much to redefine. I think ideally it should all be a bit more decoupled, but because of the use of the `FormControl` component and the way it couples with the input configs, I can't see a way of doing it that feels satisfying! 

Separately defining the columns would work though, but as the format of the data you're getting is still determined by the `formInputConfig`, so you're still kind of coupled with it. But I think you're right that it's worth doing. Maybe something to be included in the issue for updating constants?

But